### PR TITLE
Creating CS1751

### PR DESCRIPTION
**Pull request by balazsberes**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by balazsberes** [comment]
Hi @BillWagner ,
How can I fix this OPS status checker? In VS, I can't see any errors/warnings. I can't see anything relevant on this page too.

Thanks,
Balazs

File: docs/csharp/misc/cs1751.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
```
Comment (Youssef1313) [inline docs/csharp/misc/cs1751.md:1]: @BillWagner Is this the correct directory for documenting new compiler diagnostics?

Comment (balazsberes) [reply docs/csharp/misc/cs1751.md:1]: You are right, I missed the folder. Changing it now. Thank you!

**Review by BillWagner — APPROVED** [review]
Thanks @balazsberes 

This is ready to merge.  I'll :shipit: now.

**Pull request merged**